lib/WebService/MinFraud/Record/Device.pm

```diff
@@ -74,17 +80,10 @@ This is the date and time of the last sighting of the device on the specified
 IP address for your user account. The string is formatted in the ISO 8601
 combined date and time in UTC.
 
-=head2 session_age
-
-A floating point number. The number of seconds between the creation of the
-user's session and the time of the transaction. Note that C<session_age> is not
-the duration of the current visit, but the time since the start of the first
-visit.
+=head2 local_time
 
-=head2 session_id
-
-A string up to 255 characters in length. This is an ID which uniquely
-identifies a visitor's session on the site.
+This is the date and time of the transaction at the UTC offset associated with
+the device. The string is in the ISO 8601 combined date and time format.
```

Comment:
I am not sure why the Perl API documents these as ISO 8601. We document the times as RFC 3339 in the other APIs and in the dev docs. ISO 8601 isn't wrong, but it is much less specific.

Reply:
True. I will make a small PR.